packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -87,6 +89,15 @@ class Neo4jGraphQL {
                 this.schema = addResolversToSchema(this.schema, resolvers);
             }
         }
+
+        if (schemaDirectives) {
+            SchemaDirectiveVisitor.visitSchemaDirectives(
+                this.schema,
+                schemaDirectives
+                // (schemaDirectives as unknown) as Record<string, typeof SchemaDirectiveVisitor>
```

Comment:
This comment is useless?

Reply:
It is, leftover from before I fixed up the schema directives types! Removed now.